pyproject.toml

```diff
@@ -26,17 +26,14 @@ extend-ignore = [
     "D204",
     "D213",
     "D215",
-    "D400",
-    "D404",
     "D406",
     "D407",
     "D408",
     "D409",
     "D413",
 ]
-ignore = ["E501", "D107"]
+ignore = ["E501"]
 extend-exclude = ["__pycache__", "*.egg_info"]
-per-file-ignores = {"tests/*" = ["D100","D101","D102","D103","D104"]}
 
 [tool.ruff.mccabe]
 max-complexity = 10
```

Comment:
I would avoid removing lint rules until we come to an agreement with Omnivector to understand how they want to style the code. I am not attached to the rules/ignores, but I would like for something more concrete to be agreed on before we start ripping things out, _especially since we virtually have the same `pyproject.toml` file_.

Reply:
The `per-file-ignores` here are to stop the linter from complaining too much about docstrings in tests at the module/class level

Reply:
Latest commit should address this: [8899708](https://github.com/canonical/slurmd-operator/commit/8899708151557814e31f1aac8afdc1b91a8f992f)

Reply:
Nice, resolving this convo.